src/core/src/service_interfaces/StatusHandler.py

```diff
@@ -1019,24 +1028,37 @@ def __get_substatus_index(self, substatus_list_name, substatus_list):
                 return substatus_index
         return None
 
-    def __recompose_truncated_status_file(self, truncated_status_file, truncated_patches, substatus_message, substatus_index):
+    def __recompose_truncated_status_file(self, truncated_status_file, truncated_patches, count_total_errors, substatus_message, substatus_index):
         """ Recompose status file with truncated patches """
-        error_code, _ = self.__get_errors_from_substatus(substatus_msg=substatus_message)
+        error_code, errors_details_list = self.__get_errors_from_substatus(substatus_msg=substatus_message)
 
         # Check for existing errors before recompose
         if error_code != Constants.PatchOperationTopLevelErrorCode.ERROR:
             self.composite_logger.log_verbose("Patches in substatus have been truncated hence updating status to [status={0}] [PreviousErrorCode={1}]".format(Constants.STATUS_WARNING, str(error_code)))
             truncated_status_file['status']['substatus'][substatus_index]['status'] = Constants.STATUS_WARNING.lower()      # Update substatus status to warning
 
+        truncated_msg_errors = self.__recompose_substatus_msg_errors(errors_details_list, count_total_errors)
+
         self.composite_logger.log_verbose("Recompose truncated substatus")
-        truncated_substatus_message = self.__update_patches_in_substatus(substatus_msg=substatus_message, substatus_msg_patches=truncated_patches)
+        truncated_substatus_message = self.__update_patches_in_substatus(substatus_msg=substatus_message, substatus_msg_patches=truncated_patches, substatus_msg_errors=truncated_msg_errors)
 
         truncated_status_file['status']['substatus'][substatus_index]['formattedMessage']['message'] = json.dumps(truncated_substatus_message)
         return truncated_status_file
 
-    def __update_patches_in_substatus(self, substatus_msg, substatus_msg_patches):
-        """ update the substatus message patches """
+    def __recompose_substatus_msg_errors(self, errors_details_list, count_total_errors):
```

Comment:
Rename to __recompose_truncated_substatus_msg_errors

Reply:
done